lib/kvdb/kvdb_rparams.c

```diff
@@ -977,6 +977,27 @@ static const struct param_spec pspecs[] = {
             },
         },
     },
+    {
+        .ps_name = "durability.size_bytes",
+        .ps_description = "durability size in bytes",
+        .ps_flags = PARAM_FLAG_EXPERIMENTAL,
+        .ps_type = PARAM_TYPE_U32,
+        .ps_offset = offsetof(struct kvdb_rparams, dur_sz_bytes),
+        .ps_size = PARAM_SZ(struct kvdb_rparams, dur_sz_bytes),
+        .ps_convert = param_default_converter,
+        .ps_validate = param_default_validator,
+        .ps_stringify = param_default_stringify,
+        .ps_jsonify = param_default_jsonify,
+        .ps_default_value = {
+            .as_uscalar = HSE_WAL_DUR_SZ_DFLT,
+        },
+        .ps_bounds = {
+            .as_uscalar = {
+                .ps_min = HSE_WAL_DUR_SZ_MIN,
+                .ps_max = HSE_WAL_DUR_SZ_MAX,
+            },
+        },
+    },
```

Comment:
I find it really obnoxious that the param name is different from the param variable name which is different from the min/max/dflt macro names.  Multiplied by all our params and it's truly a mess IMO...

Reply:
Okay, can change the variable name to "dur_size_bytes" and the macro to "HSE_WAL_DUR_SIZE_BYTES_DFLT/MIN/MAX..."